a3fe/run/calculation.py

```diff
@@ -210,6 +214,7 @@ def setup(
                 stream_log_level=self.stream_log_level,
                 slurm_config=self.slurm_config,
```

Comment:
Should be a copy, rather than pointing to the same object.

Reply:
Just use slurm_config = self.slurm_config.copy()

So we should think about how to update the options recursively for all simulation runners in future.